File: app/src/main/java/booboo/thelocalnick/gettingstarted/testclass.java
package booboo.thelocalnick.gettingstarted;

import android.view.View;
import android.widget.Toast;

/**
 * Created by AshwinKumar on 2/8/17.
 */

public class testclass {
    public View.OnClickListener onCommentsClicked() {
        return new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                Toast.makeText(view.getContext(), "Opens comments detail", Toast.LENGTH_SHORT).show();
            }
        };
    }
    //ActivityGettingStartedBinding binding = DataBindingUtil.setContentView(this, R.layout.activity_getting_started);

}

File: app/src/main/java/booboo/thelocalnick/gettingstarted/testclass.kt
package booboo.thelocalnick.gettingstarted

import booboo.thelocalnick.models.Tour
import io.reactivex.Observable
import retrofit2.http.GET
import retrofit2.http.Query

/**
 * Created by AshwinKumar on 2/8/17.
 */

interface testclass {
    @GET("search?")
    fun getTours(@Query("location") location: String): Observable<Tour>
}